README.md

```diff
@@ -87,7 +87,8 @@ The directory structure of this repository is as follows:
 # Building
 
 It is recommended to use [`nix`](https://nixos.org/nix/download.html) for building everything in this repository.
-Haskell files can be built with [`cabal`](https://www.haskell.org/cabal/) inside of a nix shell.
+Haskell files can be built with [`cabal`](https://www.haskell.org/cabal/) inside of a nix shell, which can be
+entered by invoking the command `nix develop` from the root directory.
 
 Make sure you have a recent version of `nix` by following this [guide](https://nixos.org/manual/nix/stable/installation/upgrading.html)
```

Comment:
This is just a nitpick, so feel free to ignore it: To me it seems like  the last sentence `Make sure you have a recent version of..` makes more sense before the instruction about building Haskell files via nix shell, because it's more closely related to the sentence before about nix in general.